src/main/webapp/css/srp_style.css

```diff
@@ -37,33 +37,68 @@ h1 {
 
 .card {
   margin-bottom: 20px;
-  padding: 24px;
+  padding: 24px 16px 24px 24px;
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  max-height: 260px;
+  overflow: scroll;
+  font: 16px Roboto, Arial, sans-serif;
+  margin-top: 20px;
 }
 
-.card .card-title {
-  font: 300 20px Roboto, Arial, sans-serif;
+.card::-webkit-scrollbar {
+  width: 8px;
+  height: 8px;
+  background-color: #FAFAFA;
 }
 
-.card .card-description {
-  font: bold 16px Roboto, Arial, sans-serif;
-  margin-top: 20px;
+.card::-webkit-scrollbar-thumb {
+  background: lightgrey;
+  border-radius: 10px;
+}
+
+.card::-webkit-scrollbar-thumb:hover {
+  background: grey;
+}
+
+.card .card-title {
+  font: 300 20px Roboto, Arial, sans-serif;
 }
 
 .card code {
-  margin-top: 10px;
-  margin-bottom: 10px;
+  margin: 10px 5px 10px 5px;
   display: block;
+  font: 12px "Lucida Console", Courier, monospace;
 }
 
 .card p {
   margin-top: 0;
   display: block;
 }
 
+pre {
+  background: #EFF0F1;
+  border-radius: 2px;
+  overflow: scroll;
+}
+
+pre::-webkit-scrollbar {
+  width: 5px;
+  height: 5px;
+  background: #FAFAFA;  
+}
+
+pre::-webkit-scrollbar-thumb {
+  background: lightgrey;
+  border-radius: 10px;    
+}
+
+pre::-webkit-scrollbar-thumb:hover {
+  background: grey; 
+}
+
```

Comment:
This code seems familiar...can you combine it? Either in one rule, or make this a separate class.